### ISSUE TITLE
ENOENT error coming while using winston-daily-rotate-file with node 

### ISSUE THREAD

**Issue by mohit3081989** [winstonjs/winston-daily-rotate-file#186]
Hi,

I have been using `"winston": "^2.4.0"` and `"winston-daily-rotate-file": "^3.3.2"`. But sometimes I receive ENOENT exceptions as below 
`{"errno":-2,"code":"ENOENT","syscall":"open","path":"/usr/src/app/logs/review-service/logs-run/6131bb6ea7b3_2018-10-18_07-58-15/xyz-2018-10-18-07.log"}`

This is the transport I am using  : 

``` javascript
const produceEventLogger = new(winston.Logger)({
    levels: {'xyz': 0},
    transports: [
        new (winston.transports.DailyRotateFile)({
            level: 'xyz',
            filename: LOG_PATH + '/xyz-%DATE%.log',
            datePattern: 'YYYY-MM-DD-HH',
            zippedArchive: true,
            maxSize: 1024 * 1024 * 200,
            maxFiles: 10,
            json: true
        })
    ]
});
```

And this is how directory looks like : 

```
[ec2-user@ip]$ ls -l
-rw-r--r-- 1 root root        0 Oct 23 05:34 xyz-2018-10-18-07.log.gz
-rw-r--r-- 1 root root     2051 Oct 23 05:40 xyz-2018-10-23-05.log
```

**Comment by mohit3081989** [comment]
Nope, I am starting my server through plain node command only.

**Comment by mattberther** [comment]
Merged #185 and pushed to npm as `winston-daily-rotate-file@3.3.4`. Please try with this version as it performs some verification before attempting to gzip. Re-open the issue if it persists, please.
